src/System.Security.Cryptography.Csp/src/project.json

```diff
@@ -1,27 +1,29 @@
 {
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.7": {
       "imports": [
-        "dotnet5.4"
+        "dotnet5.8"
       ],
       "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.0.1",
-        "System.Diagnostics.Contracts": "4.0.0",
-        "System.Diagnostics.Debug": "4.0.10",
-        "System.IO": "4.0.10",
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.20",
-        "System.Runtime.Extensions": "4.0.10",
-        "System.Runtime.InteropServices": "4.0.20",
-        "System.Security.Cryptography.Algorithms": "4.2.0",
-        "System.Security.Cryptography.Encoding": "4.0.0",
-        "System.Text.Encoding": "4.0.10",
-        "System.Threading": "4.0.10"
+        "Microsoft.NETCore.Platforms": "4.3.0-beta-devapi-24512-01",
+        "System.Diagnostics.Contracts": "4.3.0-beta-devapi-24512-01",
+        "System.Diagnostics.Debug": "4.3.0-beta-devapi-24512-01",
+        "System.IO": "4.3.0-beta-devapi-24512-01",
+        "System.Reflection": "4.3.0-beta-devapi-24512-01",
+        "System.Resources.ResourceManager": "4.3.0-beta-devapi-24512-01",
+        "System.Runtime": "4.3.0-beta-devapi-24512-01",
+        "System.Runtime.Extensions": "4.3.0-beta-devapi-24512-01",
+        "System.Runtime.InteropServices": "4.3.0-beta-devapi-24512-01",
+        "System.Security.Cryptography.Algorithms": "4.3.0-beta-devapi-24512-01",
+        "System.Security.Cryptography.Encoding": "4.3.0-beta-devapi-24512-01",
+        "System.Text.Encoding": "4.3.0-beta-devapi-24512-01",
+        "System.Threading": "4.3.0-beta-devapi-24512-01",
+        "System.Threading.Tasks": "4.3.0-beta-devapi-24512-01"
```

Comment:
FYI the new references to Reflection and Threading.Tasks are needed to avoid ambiguous type compile error msgs